app/models.py

```diff
@@ -2324,3 +2324,24 @@ class FeatureToggle(db.Model):
 
     def serialize(self):
         return {"name": self.name, "is_enabled": self.is_enabled, "display_html": self.display_html}
+
+
+class FailedLoginCountByIP(db.Model):
```

Comment:
Actually, further to the comment below, should this name be changed to 'FailedLogin'?
This would follow the convention that other structures follow, of table name -> "failed_logins" with class "FailedLogin"